app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager.rb

```diff
@@ -25,7 +25,8 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::NetworkManager < Manag
            :hostname,
            :default_endpoint,
            :endpoints,
-           :snapshots,
```

Comment:
I bet we noticed this issue back when snapshots were added, but didn't realize the scope was bigger than just snapshots.

Reply:
:+1: yeah I think you're right about that